pkg/client/chains.go

```diff
@@ -0,0 +1,133 @@
+package client
```

Comment:
Renamed the file to `chains.go` from `networks.go`. Too bad it's showing it as a new file.